### ISSUE TITLE
Feature Request: Get file creation from git and use it as Copyright-Year

### ISSUE THREAD

**Issue by jonasstein** [fsfe/reuse-tool#613]
It would be very handy to create the Copy-Right-Year automagical from the local git repository with a new parameter like
`--get-year-from-git`.

We assume here that year when staged added to git may be used as year of creation.

**Comment by mxmehl** [comment]
Yeah, we discussed that a number of times, but decided against adding it as a feature of the tool for various reasons.

However, we [documented some neat oneliners](https://reuse.readthedocs.io/en/latest/scripts.html#year-of-first-commit) that would make this quite easy:

```sh
reuse addheader \
  -l Apache-2.0 \
  -c $(git log --reverse --date="format:%Y" --format="format:%cd" $FILENAME | head -n 1) \
  $FILENAME
```

With this as hand, you should quite easily be able to run this over one or many files in a repo, or add it to a pre-commit hook if you wanted.
